<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widget_cliente</class>
 <widget class="QWidget" name="widget_cliente">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>349</width>
    <height>266</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="boton_guardar">
     <property name="text">
      <string>Guardar</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="boton_eliminar">
     <property name="text">
      <string>Eliminar Cliente</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_1">
         <property name="text">
          <string>Nombre</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Apellido</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Dirección</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Teléfono</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Cubiertos</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <layout class="QVBoxLayout" name="cubiertos"/>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="nombre"/>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="apellido"/>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="dire"/>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="tel"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
